### ISSUE TITLE
[WebGL] Fix WebGL build on master

### ISSUE THREAD

**Issue by JoshEngebretson** [AtomicGameEngine/AtomicGameEngine#1345]
We have PR coverage on Windows, macOS, Linux, Android, and iOS, however WebGL currently does not, it appears I broke the build!

Here's at least the first error when compiling under Emscripten

```
/Users/jenkins/Home/workspace/Atomic_EditorWebPlayer/Source/Atomic/Metrics/Metrics.cpp:97:9: error: use of undeclared identifier 'snprintf'; did you mean 'vswprintf'?
        snprintf(name, NAME_MAX_LENGTH, "%-20s", metric.classname.CString());
```

There may be more errors, or this may be the only one, who will find out? 🎱

**Comment by JimMarlowe** [comment]
I am able to reproduce this, it appears to be a header file issue with emcc.

**Comment by JoshEngebretson** [comment]
Closed via #1347
